pkg/security/podsecuritypolicy/util/util.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 const (
-	ValidatedPSPAnnotation = "kubernetes.io/psp"
```

Comment:
I wouldn't change the existing annotation

Reply:
fixed